### PR TITLE
2702 refactor/sample data for better

### PR DESCRIPTION
**Pull request by perryr16**
#### Any background context you want to provide?
The homepage has a feature allowing users to "Auto-Populate Sample Data". This generates 12 sample properties, one of which allows a user to run a BSync Analysis. At this time there is no sample property that allows users to run a BETTER Analysis.

#### What's this PR do?
- Adds a sample building to be used in a BETTER analysis ('Campus Building 2')
- Adds corresponding meter data to 'PM Meter Data.xlsx'. This data was taken from existing JCC building meters and modified. The Electrical usage data has been modified by a random percentage between +/- 15%, while the natural gas usage data has been modified by a random percentage between -3% and 3%
- Modifies the auto-populate modal to have instructions for a sample BETTER analysis
- Modifies the auto-populate modal style to encourage users to scroll down as the instructions extend beyond the viewing height of most screens.

#### How should this be manually tested?
On the homepage, select 'Auto-Populate Sample Data'. There should now be additional information under the 'BETTER' category. If the continue button is grayed out, follow instructions to auto-populate the sample data. 
Click Continue.
Go to Inventory.
Select building with PM Property ID = `5766981` and choose to run a BETTER analysis. The results should show examples of `Electricity Consumption` and `Fossil Fuel Consumption

#### What are the relevant tickets?
#2702 

#### Screenshots (if appropriate)
Gradient prompts users to scroll downward.
Screen Shot 2021-11-05 at 1 44 45 PM
Screen Shot 2021-11-05 at 1 44 51 PM

BETTER Analysis for sample Building 5766981
Screen Shot 2021-11-05 at 2 30 07 PM
Screen Shot 2021-11-05 at 2 30 27 PM

### CODE REVIEW

File: seed/tests/data/property_sample_data.json [hunk 1]
```diff
@@ -225,4 +225,27 @@
   "owner_city_state": "Colorado",
   "owner_postal_code": 80202,
   "organization_id": -1
+}, {
+  "jurisdiction_property_id": 123481,
+  "pm_parent_property_id": "5766981",
+  "pm_property_id": "5766981",
+  "property_type": "Office",
```
Comment (perryr16) [inline seed/tests/data/property_sample_data.json:232]: Any logic to assigning the IDs? the pm_property_id is incremented by 2 from the sample property above

File: seed/views/v3/organizations.py [hunk 2]
```diff
@@ -1510,7 +1510,7 @@ def insert_sample_data(self, request, pk=None):
         import_record = ImportRecord.objects.create(name='Auto-Populate', super_organization=org)
 
         # Interval Data
-        filename = 'PM Meter Data 12.xlsx'
+        filename = 'PM Meter Data.xlsx'  # contians meter data for BSync and BETTER
```
Comment (macintoshpie) [inline seed/views/v3/organizations.py:1513]: Could you delete the old file?

```suggestion
        filename = 'PM Meter Data.xlsx'  # contians meter data for bsyncr and BETTER
```

File: seed/tests/data/property_sample_data.json (hunk 1 above)
Comment (macintoshpie) [reply seed/tests/data/property_sample_data.json:232]: I think that's fine

File: seed/tests/data/property_sample_data.json [hunk 3]
```diff
@@ -225,4 +225,27 @@
   "owner_city_state": "Colorado",
   "owner_postal_code": 80202,
   "organization_id": -1
+}, {
+  "jurisdiction_property_id": 123481,
```
Comment (macintoshpie) [inline seed/tests/data/property_sample_data.json:229]: Your new property is getting merged with another one...
Screen Shot 2021-11-08 at 9 43 45 AM

According to the "autopopulate modal" the two properties above your new one are supposed to be merged. Could you check the logic in the endpoint to make sure we're merging the correct properties? My guess is it's just grabbing the last two properties but that wouldn't be right now

**Review by macintoshpie — CHANGES_REQUESTED** [review]
Tested the analysis on the new property and it works! Just a few small requests

**Review by macintoshpie — APPROVED** [review]
💯  Nice!

**Pull request merged**